bagisto/bagisto
# Billing Address Formatted Issue #5337 Fixed

## Issue Reference
#5337 

## Description
Billing Address Formatted 

## How To Test This?
1. Go to admin -> Sales -> Orders -> Actions
2. Now look at the invoice.
3. Check the address details.
